scripts/create_registry_schema.py

```diff
@@ -261,13 +273,18 @@ def _Keyword(schema):
     columns = _get_column_definitions(schema, "keyword")
 
     # Table metadata
-    meta = {"__tablename__": "keyword", "__table_args__": (UniqueConstraint(
-                "keyword", name="keyword_u_keyword"
-            ), {"schema": schema},)}
+    meta = {
+        "__tablename__": "keyword",
+        "__table_args__": (
+            UniqueConstraint("keyword", name="keyword_u_keyword"),
```

Comment:
All the information about constraints should be read from `schema.yaml`.  Currently it looks like the constraint name (here "keyword_u_keyword") does not appear in `schema.yaml`.  It should.  And the code should get it from that file. We don't want that kind of information in two places.  
Looking more closely, I think this is already being handled properly.  `_Keyword` is not being called anywhere.  I think it and `_DatasetKeyword` can just be removed.